prow/config.yaml

```diff
@@ -6824,7 +6824,52 @@ periodics:
         secretName: ssh-key-secret
     - name: var-lib-docker
       emptyDir: {}
-
+- name: ci-ingress-gce-e2e
+  agent: kubernetes
+  interval: 60m
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20180108-bf63630a9-master
+      args:
+      - "--repo=k8s.io/ingress-gce=$(PULL_REFS)"
+      - "--root=/go/src/"
+      - "--upload=gs://kubernetes-jenkins/pr-logs"
+      - "--clean"
```

Comment:
oh hummmm, you'll need a `--timeout=110` flag here (anything greater than 90m+20m)

cc @BenTheElder that presubmit is definitely broken :face_with_head_bandage:

Reply:
Done